packages/shared/src/components/ShareOptionsMenu.tsx

```diff
@@ -47,10 +47,10 @@ export default function ShareOptionsMenu({
   onHidden,
   contextId = 'share-context',
 }: ShareOptionsMenuProps): ReactElement {
+  const { openNewSquadModal } = useCreateSquadModal();
```

Comment:
I think we should merge this PR into https://github.com/dailydotdev/apps/pull/1944 instead, since there is additional cleanup there that was started and pending. It will raise more conflicts like this. We also have `useSquadNavigation` hook there that can also potentially be removed with `useCreateSquadModal` all together.

Reply:
We want to merge this to main today, that's kind of the constraint for now.
I have another one open to merge it into squads-mvp, so will adjust once that's done.